Source/Csla/Serialization/Mobile/MobileFormatter.cs

```diff
@@ -117,6 +111,15 @@ private void ConvertEnumsToIntegers()
       }
     }
 
+    private MobileFormatterOptions GetOptions()
+    {
+      var cslaOptions = applicationContext.GetRequiredService<CslaOptions>();
+      var options = (MobileFormatterOptions)cslaOptions.SerializationOptions.FormatterOptions;
+      if (options is null)
+        throw new ArgumentNullException("MobileFormatterOptions");
```

Comment:
imho should be `InvalidOperationException` because it's not an argument anywhere.